specs/pvs/corla.pvs

```diff
@@ -1420,8 +1457,71 @@ BEGIN
   number_of_ballots_to_audit: [audited_contest -> nat]
 
 %%% \todokiniry{Still need to model core RLA algorithm(s).}
+
+% @trace RDPR-6-Jul-2017 Section 25.1.4
+% @description A type of risk-limiting audit in which the audit board
+% examines and reports to the secretary of state voter markings on
+% randomly selected ballots, and then compared compares them to the
+% voting system's tabulation as reflected in the corresponding cast
+% vote records.
+  comparison_audit: TYPE FROM risk_limiting_audit
 END rlas
 
+%%% In order to instantiate the general idea of Risk-Limiting Audits
+%%% into the context of Colorado elections, we must: (1) choose RLA
+%%% parameters appropriate for Colorado's context, and (2) instantiate
+%%% the operation of the concurrent RLA across all counties in a
+%%% fashion that fits the needs of the CDOS.
+
+colorado_rlas: THEORY
+BEGIN
+  IMPORTING rlas
+
+%%% The total number of ballot cards uploaded across the entire state.
+  total_ballot_cards: nat
+
+%%% Next, we'll formalize what is stated in the current draft
+%%% rules. First, the risk limit for comparison audits must be below
+%%% $5\%$.
+% @trace RDPR-6-Jul-2017 Section 25.2.2(A)
+  RDPR_risk_limit: TYPE = {n : nonneg_real | n <= 0.05}
```

Comment:
I still need to model what is in the statutes and in the rules.  That's doesn't mean that we will implement these dependent types as invariants in the system.